test/drivers/test_suites/syscall_exit_suite/execveat_x.cpp

```diff
@@ -174,7 +174,6 @@ TEST(SyscallExit, execveatX_correct_exit)
 	/* Prepare the execve args */
 	int dirfd = 0;
 	const char *pathname = "/usr/bin/echo";
-	const char *comm = "echo";
```

Comment:
This line should not be removed as this is required below for the `__s390x__` block. However, I have corrected this with PR #1150. Thanks.

Reply:
oh, bad trip hahaha! thank you for the hint